src/graphql/controller.ts

```diff
@@ -205,6 +205,10 @@ export class GqlEntityController {
   public async createEntityStores(knex: Knex): Promise<{ builder: Knex.SchemaBuilder }> {
     let builder = knex.schema;
 
+    if (knex.client.config.client === 'pg') {
```

Comment:
Do we support other db than Postgres? Or what is this condition for?

Reply:
At the moment just for our tests as those run with SQLite.